src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -98,6 +97,9 @@ class GcsServer {
   /// Check if gcs server is stopped.
   bool IsStopped() const { return is_stopped_; }
 
+  /// Retrieve cluster ID
+  const ClusterID GetClusterId() const { return rpc_server_.GetClusterId(); }
```

Comment:
const & ?

Reply:
I'll change it, but with RVO it shouldn't matter very much